### PR TITLE
Tbl109 pipelines

### PR DESCRIPTION
**Pull request by benjefferies**
Add manifest file for cloudfoundry deployments

1. Are the defaults set to sensible values?
1. Are we missing any manifest properties that should be there?
1. Could we strip them down further?

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+applications:
+- name: ras-backstage
+  disk_quota: 1G
```
Comment (LJBabbage) [inline manifest.yml]: At present prod is on 512M

File: manifest.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  health-check-http-endpoint: /info
+  health-check-type: http
+  instances: 1
```
Comment (LJBabbage) [inline manifest.yml]: We are running 3 instances on prod

File: manifest.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  memory: 512M
```
Comment (LJBabbage) [inline manifest.yml]: Prod memory: 256M

File: manifest.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  stack: cflinuxfs2
```
Comment (LJBabbage) [inline manifest.yml]: Stack?

**Comment by LJBabbage** [comment]
Although I realize these are the defaults for SIT, I think we need to start aligning them with PROD/PREPROD

File: manifest.yml (hunk 4 above)
Comment (benjefferies) [reply manifest.yml]: All of the configuration was generated from the cloudfoundry command line tool. `cf create-app-manifest`. I can update the manifests to reflect prod if we feel like it's necessary right now but as jenkins doesn't use the manifest I'm not sure it's worth it

**Pull request merged**